README.md

```diff
@@ -210,6 +210,9 @@ Users can bootstrap the Windows instance without the need to provide the user-da
 
 ***Note***: Users also need to pass the `--security-group-ids` option with IDs of the security group(s) having the required ports opened like `5986` for winrm ssl transport. In case if `--security-group-ids` option is not passed then make sure that the default security group in your account has the required ports opened.
 
+Users also need to pass the `--chef-node-name` option with node name, user can also append ec2 server id by appending `%s` into node name. The %s replaced by the ec2 server id dynamically. 
+e.g. `-N "TestCLI%s" or  --chef-node-name "TestCLI%s"`
+
```

Comment:
This can be put at some place where it's easily visible to the user. It can look like this:

**Appending server_id to the node name**
Users can also append ec2 server_id to the node name by appending `%s` into `--chef-node-name` option. The %s is replaced by the ec2 server id dynamically. 
e.g. `-N "<node-name>%s" or  --chef-node-name "<node-name>%s"`